js/templates/accountsettings.html

```diff
@@ -0,0 +1,16 @@
+<div class="section" id="account-info">
+	<h2>Account Settings - {{ email }}</h2>
+</div>
+
+<div class="section" id="aliases">
+	<h2 class="icon-loading">Aliases</h2>
+	<div id="aliases-table"></div>
+	<form name="aliasForm" method="post">
+		<input type="text" name="alias" id="alias" placeholder="add new alias">
+		<input type="submit" value="Save">
+		<span id="aliasSavingMsg" class="msg hidden"> saving </span>
+	</form>
+</div>
+<div class="section" id="signature">
+	<h2>Signature</h2>
```

Comment:
oh, that section is actually out of scope for this PR. Please remove it so we can have a release after you finished the alias feature, but before you work on the signature feature ;-)